webapp/src/App.js

```diff
@@ -168,6 +169,13 @@ class EventNav extends Component {
                   >
                     {t('Review Dashboard')}
                   </NavLink>
+                  <NavLink
+                    to={`/${this.props.eventKey}/tagConfig`}
```

Comment:
Ah this will try to navigate to something like /indaba2023/tagConfig, but your route below is just defined for /tagConfig (ie at the organisation level). Since we do want this page to be at the event level, you should define the route in [EventHome.js](https://github.com/deep-learning-indaba/Baobab/blob/develop/webapp/src/pages/eventHome/EventHome.js#L128) where the `${match.path}` in the path will match the event key :)

Reply:
Resolved, thanks!